### ISSUE TITLE
Issue for MP04

### ISSUE THREAD

**Issue by cndy17** [cndy17/STA9750-2024-FALL#5]
Hi @michaelweylandt!

I've created my MP04 website - check it out!

https://cndy17.github.io/STA9750-2024-FALL/mp04.html

**Comment by cliu98** [comment]
Hi @cndy17 
Here's the breakdown of my feedback:

Written Communication: 10/10
Your report was well written and communicated your findings effectively. Your combination of using bullet points to separate the different plans while communicating their respective results was well thought out and easy to read. 

Project Skeleton: 10/10 
All tasks are completed with insightful commentary.

Formatting & Display: 8/10
All graphs and tables are meaningful and labeled properly.  I only left points out as the rubric required an animated visualization for full score. 

Code Quality: 10/10
Your code is organized well, easy to read and understand. 

Data Preparation: 10/10
All the data is prepared in an organized and structured manner, cleaning is done meaningfully.  

Overall: 48/50
I think you did an amazing job on your report. It was well articulated and I had no issues understanding what information you were trying to communicate with the data. Everything included was meaningful to either add context or help draw conclusions.

**Comment by camilomartinez777** [comment]
Hi @cndy17 

10/10: Written Communication: I like the way you communicate the project. It has good storytelling. 
10/10: Formatting & Display:  I could see all the images perfectly. 
8/10: Projet Skeleton: This is the part where the project could worked more. You could answer the question, what is the best percentage to withdraw 4% or 6% or something else? 	
10/10: Code Quality: Explained the code, and it work well.
10/10: Data Preparation:  Everything works good.

Total 48/50:  It could be more if you add the code. But good job. 

Great job.

**Comment by laurencardieri** [comment]
Hi @cndy17! The following is my feedback. 

Written Communication: 10/10: The way you communicated in this project was very effective and informative. You left the reader with no questions and wrote a great report. Key findings are highlighted and appropriately given context. I also enjoyed how you also compared the plans at a theoretical level. I also like how you explained what each of the data sources you were taking from were. 

Project Skeleton: 10/10: I believe you fulfilled all of the professors expectations for this project and then some. All tasks were completed effectively. 

Formatting & Display: 9/10. Your displays of the information was done very intelligently, and I loved how you compared your data using line graphs. I also loved how you displayed your Monte Carlo information. The only reason I took a point off is because there was a requirement to include an animated graph. Other than that, great work! 

Code Quality: 10/10: I believe your code was done flawlessly. I was unable to find any errors or anything to improve upon. I also enjoyed how you put the hashtags in to explain what you were doing.  

Data Preparation: 10/10: You were able to download the data in without hand downloading anything. You were also able to hide your key. Great work! 

Total Score: 49/50.
